ts-defs-src/test-api/test-controller.d.ts

```diff
@@ -98,6 +98,19 @@ interface WindowFilterData {
 
 type ScrollPosition = 'top' | 'right' | 'bottom' | 'left' | 'topRight' | 'topLeft' | 'bottomRight' | 'bottomLeft' | 'center';
 
+interface Cookie {
```

Comment:
Let's add a public type definition after the feature is completely implemented. Now, just remove it.